nx_altair/draw_altair.py

```diff
@@ -281,6 +281,119 @@ def draw_networkx_nodes(
 
     return node_chart
 
+def draw_networkx_nodes_labels(
+    G=None,
+    pos=None,
+    chart=None,
+    layer=None,
+    nodelist=None,
+    node_label_size=15,
```

Comment:
* `node_label_size` should be `font_size`
* `node_label_color` should be `font_color`
* `node_label` should be `labels`.